### ISSUE TITLE
Tokenizer throws an exception when created with Llama3 

### ISSUE THREAD

**Issue by nmetulev** [microsoft/onnxruntime-genai#281]
Repro:
1. Build llama3 for cpu int4 using the model builder
1. Use model in a .NET Core app (I'm using .NET Core 8 and WinAppSdk) 
	```csharp
	_model = new Model(".\\modelPath\\llama3");
	_tokenizer = new Tokenizer(_model);
	```
1. Run app 

The following excpetion is thrown:

```
Microsoft.ML.OnnxRuntimeGenAI.OnnxRuntimeGenAIException
  HResult=0x80131500
  Message=NO_SUCH_FIELD: The JSON field referenced does not exist in this object.
  Source=Microsoft.ML.OnnxRuntimeGenAI
  StackTrace:
   at Microsoft.ML.OnnxRuntimeGenAI.Result.VerifySuccess(IntPtr nativeResult)
   at Microsoft.ML.OnnxRuntimeGenAI.Tokenizer..ctor(Model model)
   at AutoComplete.Autocomplete.LlamaOrtGenAIAutocomplete.<LoadModelAsync>b__9_0() in F:\source\AutoComplete\AutoComplete\Autocomplete\LlamaOrtGenAIAutocomplete.cs:line 43
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
```

**Comment by natke** [comment]
We have a fix for this in progress @nmetulev

**Comment by natke** [comment]
This should be fixed in the upcoming 0.2.0-rc7 release

**Comment by natke** [comment]
Closing as fixed
